kore/src/Kore/Builtin.hs

```diff
@@ -266,3 +271,16 @@ externalizePattern =
                 $ externalizePatternWorker
                 $ getEvaluated evaluatedF
             BuiltinF _ -> error "Unexpected internal builtin"
+
+internalize
+    :: (Ord variable, SortedVariable variable)
+    => SmtMetadataTools Attribute.Symbol
+    -> TermLike variable
+    -> TermLike variable
+internalize tools =
+    Recursive.fold (internalize1 . Recursive.embed)
+  where
+    internalize1 =
+            List.internalize tools
+        >>> Map.internalize tools
```

Comment:
Am I misunderstanding this, or is `>>>` used here as `flip (.)`? And since they're all essentially `a -> a`, does the order matter? Could it not be replaced by `.`?

Reply:
You understand correctly. I did not have a particular reason for using it here.